esmvaltool/recipes/recipe_collins13ipcc.yml

```diff
@@ -0,0 +1,2539 @@
+###############################################################################
+# recipe_collins13ipcc.yml
+
+---
+documentation:
+  description: | 
+    Reproducing selected figures from IPCC AR5, chap. 12 (Collins et al., 2013)
+
+  authors:
+   - lore_ru
+
+  references:
+   - collins13ipcc
+
+  projects:
+    - crescendo
+
+preprocessors:
+  preprocessor_1:
```

Comment:
You do not need to specify a preprocessor if you only need the default preprocessor settings, so I think you can remove `preprocessor_1`. There is also no need to set `regrid`, `mask_landsea` and `multi_model_statistics` to `false`, because this is the default, so this can also be removed in the preprocessors below.